tags.go

```diff
@@ -32,6 +31,10 @@ type Tag struct {
 	Value string `json:"value"`
 }
 
+func (t Tag) HasSameKeyValue(otherTag Tag) bool {
```

Comment:
why not just `Equals` for the function name?

Reply:
Because this does not account for the `Id`

Reply:
When somebody provides "tags wanted" for creation, the Id will not be known for those tags